easybuild/easyconfigs/g/gmsh/gmsh-4.11.1-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.11.1'
+
+homepage = 'https://gmsh.info/'
+description = "Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://%(name)s.info/src/']
+sources = ['%(name)s-%(version)s-source.tgz']
+checksums = ['c5fe1b7cbd403888a814929f2fd0f5d69e27600222a18c786db5b76e8005b365']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('SWIG', '4.0.2'),
+    ('Eigen', '3.4.0'),
+]
+dependencies = [
+    ('Python', '3.10.4'),
+    ('PETSc', '3.17.4'),
+    ('SLEPc', '3.17.2'),
+    ('FLTK', '1.3.8'),
+    ('occt', '7.5.0p1'),
+]
+
```

Comment:
I saw there used to be `separate_build_dir` in previous versions of this EasyConfig. Did you take that out on purpose?

Reply:
Er, no, it seems to have been dropped with the `--try-*`-automation. As the tests succeed, I didn't notice it had gone. I'll put it back in. Mental note, more checking on the output of `--try-*`.

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17534/commits/f7cb9a083ab4b051084c37027ff0839b76c65500

Reply:
Yeah, the `--try` things are awesome, but sometimes awkward. I knew they did template replacement, but I'm surprised they also drop keywords! Good to know...